Dockerfile.dev

```diff
@@ -1,4 +1,16 @@
-FROM nginx:alpine
-COPY dist/ /usr/share/nginx/html/
-COPY .docker/nginx.conf /etc/nginx/conf.d/default.conf
-COPY .docker/mime.types /etc/nginx/mime.types
\ No newline at end of file
+# Build final image
+FROM node:14-alpine
+
+WORKDIR /app
+
+COPY .docker/package.json .docker/nuxt.config.js ./
+
+# Copy client files from the build image
+COPY dist/ ./.nuxt/
+
+# Install runtime dependencies
+RUN yarn install --production
+
+EXPOSE 80
```

Comment:
If we're going to bother with a useless line only good for documentation, might as well use a `VOLUME` directive as well to persist state even when someone forgets to create a volume.

Reply:
These are changes from the SSR PR, not this one :p I'm only basing this branch on that PR to make sure it works correctly in both cases ;)

Reply:
> If we're going to bother with a useless line only good for documentation, might as well use a `VOLUME` directive as well to persist state even when someone forgets to create a volume.

I know this is a comment not relevant to this PR, however the `VOLUME` instruction in Dockerfiles **is not** just a documentation instruction. IF used this it will **guarantee** that a volume is bound to the location specified and will end up creating a anonymous-volume if there was no bind for the location.
There are also other implication that this instruction will lead to, such as that the path specified in the `VOLUME` array can no longer be modified by any following instruction of the Dockerfile or any subsequent Dockerfiles using it as a dependency.

(sorry felt the need to specify this)